### ISSUE TITLE
Related collections picker shrinks with content

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/directus#19516]
### Describe the Bug

<img width="274" alt="CleanShot 2023-08-23 at 16 36 15@2x" src="https://github.com/directus/directus/assets/9141017/0b18ebdd-7942-45ce-8dc3-dff3c4918da8">

This should be `width` instead of `max-width`

### To Reproduce

Choose related collections, search for a value that's smaller than the width of the field.

### Directus Version

v10.5.3

### Hosting Strategy

Directus Cloud

**Comment by DanielBiegler** [comment]
Managed to make it at least as wide as the `reference` which is an improvement but not perfect.

| Before | After |
|--------|--------|
| [Screencast from 24.08.2023 12:11:07.webm](https://github.com/directus/directus/assets/14810858/f92c2262-c9f6-483e-be4c-ffabb98f3e4a) | [Screencast from 24.08.2023 12:08:28.webm](https://github.com/directus/directus/assets/14810858/8255f282-a64b-4fb8-ab95-80f6dc73c483) |

**Comment by rijkvanzanten** [comment]
> Do we want to get rid off the dynamic size completely? Reading longer system tables like the future directus_extension_permissions might be painful then 😄

Nope! It should grow if longer things are displayed. The min-width should be the size of the parent attached input instead of the shorted value shown though 🙂

**Comment by rijkvanzanten** [comment]
Oh looks like your before and after is exactly what I was looking for 🙂
